lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -18,6 +18,8 @@
 description:
     - This module manages whole lifecycle of the Virtual Machine(VM) in oVirt/RHV.
     - Since VM can hold many states in oVirt/RHV, this see notes to see how the states of the VM are handled.
+    - "WARNING: If VM parameter, which requires reboot, is updated, then internally oVirt engine always create new snapshot
```

Comment:
Please move this sentence to the notes: section as the first note, and change it to say:
    -  If you update a VM parameter that requires a reboot, the oVirt engine always creates a new snapshot for the VM, and an Ansible playbook will report this as changed.